.golangci.yml

```
@@ -0,0 +1,251 @@
+    - nosprintfhostport
+    - reassign
+    - rowserrcheck
+    - sqlclosecheck
+    - thelper
+    - usestdlibvars
+    - wastedassign
+
+  disable:
+    # Disabled for CLI applications with acceptable use patterns
+    - gochecknoinits  # CLI commands use init functions for Cobra setup
+    - gosec          # File reading warnings for config loading are acceptable  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)